src/openlcb/SimpleInfoProtocol.hxx

```diff
@@ -396,23 +396,21 @@ private:
     }
 
     /** Configuration option. See constructor. */
-    unsigned maxBytesPerMessage_ : 8;
+    uint8_t maxBytesPerMessage_;
     /** Configuration option. See constructor. */
-    unsigned useContinueBits_ : 1;
+    uint8_t useContinueBits_ : 1;
 
     /** Whether this is the first reply message we are sending out. Used with
      * the continuation feature. */
-    unsigned isFirstMessage_ : 1;
+    uint8_t isFirstMessage_ : 1;
     /** Tells which descriptor entry we are processing. */
-    unsigned entryOffset_ : 5;
-#pragma clang diagnostic ignored "-Wunused-private-field"
-    unsigned reserved_ : 1; // for alignment
+    uint8_t entryOffset_ : 5;
```

Comment:
Is the alignment issue not really an issue?

Reply:
we could also drop : 5 if switching to bool for the other two single bit usages

Reply:
bool is 32 bits.
alignment is provided by the next element being a "full" type instead of a bitfield.